go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/EvergenEnergy/TimeSchema
+
+go 1.21
+
+require github.com/aws/aws-sdk-go-v2 v1.24.1
```

Comment:
not sure why this one ended on a separate `require` line but it can probably be moved inside the first block

Reply:
done